### DISCUSSION TITLE
Add python script to create IDAES-PSE data used in the docs?

### DISCUSSION THREAD

**Discussion by Maltimore** [cog-imperial/OMLT#87] [General]
Hi,
this is a bit of a long shot: I looked at your example [notebook here](https://github.com/cog-imperial/OMLT/blob/main/docs/notebooks/neuralnet/auto-thermal-reformer.ipynb), and was wondering whether you'd like to share the python script with which you generated the data? The reason is that I'd like to look at the same example process, but I need to export more intermediate values to the csv output, so I thought I might build on your script.

**Comment by rmisener** [comment]
Hi @Maltimore, thanks for asking. I switched this question from an "issue" to a "discussion" because this is a request for an additional document and not a problem with OMLT.

@carldlaird can clarify if I have something wrong, but the IDAES (https://github.com/IDAES/idaes-pse) model we used to generate that data is not ours to share. If I've understood correctly, the model itself belongs to @andrewlee94 (he may not be able to share it, either, not sure).

**Comment by Maltimore** [comment]
Hi @rmisener
thanks for the quick reply and moving this appropriately into a discussion.
Regarding the model, I wonder whether it helps that I am working indirectly for BASF, and more importantly I have a non-disclosure agreement with BASF (since I saw that this work was partially funded by BASF and they are involved somehow).

**Comment by rmisener** [comment]
No, BASF is irrelevant to this subproject. That particular model belongs to Sandia National Laboratories, not BASF. BASF kindly funded the tree portion of the project via the PhD of @ThebTron.
